File: src/main/java/no/nav/domain/pensjon/regler/beregning/penobjekter/FasteUtgifterTilleggUT.java
package no.nav.domain.pensjon.regler.beregning.penobjekter;

import no.nav.domain.pensjon.regler.beregning.Ytelseskomponent;
import no.nav.domain.pensjon.regler.kode.YtelsekomponentTypeCti;

public class FasteUtgifterTilleggUT extends Ytelseskomponent {
   private Integer nettoAkk;
    private Integer nettoRestAr;
    private Integer avkortningsbelopPerAr;

    public FasteUtgifterTilleggUT() {
        super();
        ytelsekomponentType = new YtelsekomponentTypeCti("UT_FAST_UTGIFT_T");
    }
}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning/penobjekter/FasteUtgifterTilleggUT.kt
package no.nav.pensjon.regler.domain.beregning.penobjekter

import no.nav.pensjon.regler.domain.beregning.Ytelseskomponent
import no.nav.pensjon.regler.domain.enum.YtelseskomponentTypeEnum

class FasteUtgifterTilleggUT : Ytelseskomponent() {
    var nettoAkk: Int? = null
    var nettoRestAr: Int? = null
    var avkortningsbelopPerAr: Int? = null

    override val ytelsekomponentTypeEnum: YtelseskomponentTypeEnum = YtelseskomponentTypeEnum.UT_FAST_UTGIFT_T
}